packages/server/src/lib/email.js

```diff
@@ -445,41 +475,43 @@ async function sendAsyncReportEmail(recipient, signedUrl, reportType) {
     const formattedBodyTemplate = fileSystem.readFileSync(path.join(__dirname, '../static/email_templates/_formatted_body.html'));
 
     const formattedBody = mustache.render(formattedBodyTemplate.toString(), {
-        body_title: `Your ${reportType} report is ready for download`,
+        body_title: `Your ${reportType.name} report is ready for download`,
         body_detail: `<p><a href=${signedUrl}><b>Click here</b></a> to download your file<br>Or, paste this link into your browser:<br><b>${signedUrl}</b><br><br>This link will remain active for 7 days.</p>`,
     });
 
-    const emailHTML = module.exports.addBaseBranding(
+    const emailHTML = addBaseBranding(
         formattedBody,
         {
             tool_name: 'Grants Reporter Tool',
-            title: `Your ${reportType} report is ready for download`,
+            title: `Your ${reportType.name} report is ready for download`,
             includeNotificationsLink: false,
         },
     );
 
-    await module.exports.deliverEmail({
+    await deliverEmail({
         fromName: ARPA_EMAIL_FROM_NAME,
         toAddress: recipient,
         emailHTML,
-        emailPlain: `Your ${reportType} report is ready for download. Paste this link into your browser to download it: ${signedUrl} This link will remain active for 7 days.`,
-        subject: `Your ${reportType} report is ready for download`,
+        emailPlain: `Your ${reportType.name} report is ready for download. Paste this link into your browser to download it: ${signedUrl} This link will remain active for 7 days.`,
+        subject: `Your ${reportType.name} report is ready for download`,
+        emailType: reportType.emailType,
     });
 }
 
 module.exports = {
```

Comment:
I "privatized" a number of functions that weren't being used outside this module.